src/hooks/useCds/translate.js

```diff
@@ -497,8 +497,13 @@ function mapStrideResult(patientData, patientDataMap, stridesData) {
 
   stridesPatientData.forEach(row => {
     const orderId = row['ORDER_ID'];
-    const diagnosticReport = patientDataMap.DiagnosticReport.find(
-      dr => dr.identifier.some(identifier => identifier.system === 'urn:oid:1.2.840.114350.1.13.284.3.7.2.798268' && identifier.value == orderId));
+    const diagnosticReport = patientDataMap.DiagnosticReport.find((dr) =>
+      dr.identifier.some(
```

Comment:
Are we worried at all about the diagnosticReport not having an identifier? It's not technically required, unless we're working under some profile I'm not aware of, and this could potentially throw an error if identifier is `dr.identifier` is null

Reply:
This is the profile it should be modeled off of: https://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-diagnosticreport-lab.html. You're right, it doesn't require an identifier, although we have seen one with every pilot test patient I have seen. But good catch, it's still worth checking so that this wouldn't cause an error if an identifier didn't exist. I'll add handling for this.

Reply:
Okay, I just added [Add error handling for missing DiagnosticReport elements](https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/75/commits/5a93bfe3696f6ea00afaebe9993a0b9d91c40cea). This should add handling for cases when the DiagnosticReport's `identifier` does not exist, or when some of the sub-elements within the `identifier` are not present. I added a test to check the case where the DiangosticReport has no `identifier`.